src/ContactPeople/ContactPersonForm/AddContactDetailsComponent.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FieldArray } from 'redux-form';
+
+import {
+  Button,
+  Col,
+} from '@folio/stripes-components';
+
+import EmbeddedContactForm from './EmbeddedContactForm';
+import css from './AddContactDetailsComponent.css';
+
+class AddContactDetailsComponent extends Component {
```

Comment:
I don't think we use pattern to add suffix `Component` for every component

Reply:
Updated